zgrab_schema.py

```diff
@@ -800,29 +805,23 @@
                     "compression":String(),
                 }),
             }),
-            "dh_key_exchange": SubRecord({
-                "params": SubRecord({
-                    "prime": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "generator": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_private": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
+            "key_exchange": SubRecord({
+                "ed25519sha256_params": SubRecord({
+                    "server_public": Binary(),
+                }),
+                "ecdh_params": SubRecord({
                     "server_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
+                        "x": golang_crypto_param,
+                        "y": golang_crypto_param,
                     }),
                 }),
+                "dh_params": SubRecord({
+                    "prime": golang_crypto_param,
+                    "generator": golang_crypto_param,
+                    "client_public": golang_crypto_param,
```

Comment:
This is fixed in 0de2c77b4f1e84e8fd790b038d8cf3989799917b — I still don't 100% understand how all this logging code works, but I can confirm that commit removed `client_private` from non-verbose XSSH grabs.